src/test/kotlin/com/jetbrains/snakecharm/SmkStatementMoverTest.kt

```diff
@@ -0,0 +1,253 @@
+package com.jetbrains.snakecharm
+
+import com.intellij.openapi.actionSystem.IdeActions
+import com.intellij.openapi.fileEditor.FileDocumentManager
+import com.jetbrains.python.psi.LanguageLevel
+import features.glue.SnakemakeWorld.myFixture
+
+class SmkStatementMoverTest: SnakemakeTestCase() {
+    private fun doTest() {
+        val testName: String = getTestName(true)
+
+        myFixture?.configureByFile("mover/$testName.smk")
+        myFixture?.performEditorAction(IdeActions.ACTION_MOVE_STATEMENT_UP_ACTION)
+        myFixture?.checkResultByFile("mover/" + testName + "_afterUp.smk", true)
+        myFixture?.getFile()?.let { myFixture?.getDocument(it)?.let { FileDocumentManager.getInstance().reloadFromDisk(it) } }
+        myFixture?.configureByFile("mover/" + getTestName(true) + ".smk")
+        myFixture?.performEditorAction(IdeActions.ACTION_MOVE_STATEMENT_DOWN_ACTION)
+        myFixture?.checkResultByFile("mover/" + testName + "_afterDown.smk", true)
+    }
+
+    fun testSimple() {
+        doTest()
+    }
+
+    fun testCommentUp() {
+        doTest()
+    }
+
+    fun testTryExcept() {
+        doTest()
+    }
+
+    fun testInnerIf() {
+        doTest()
+    }
+
+    fun testNestedIfUp() {
+        doTest()
+    }
+
+    fun testCommentOut() {  //PY-5527
```

Comment:
What do all these tests? Do we really need to in-depth test all python line mover use cases here?

Reply:
I removed this tests and added cucumber tests